boxoffice/models/line_item.py

```diff
@@ -107,6 +107,12 @@ def current_assignee(self):
         return self.assignees.filter(Assignee.current == True).one_or_none()
 
 
+@property
+def get_confirmed_line_items(self):
+    return LineItem.query.filter(LineItem.order == self, LineItem.status == LINE_ITEM_STATUS.CONFIRMED).all()
+Order.get_confirmed_line_items = get_confirmed_line_items
```

Comment:
Don't use the @property decorator. Instead, use `Order.get_confirmed_line_items = property(get_confirmed_line_items)`. Also, two space above.